src/data/faq_de.json

```diff
@@ -2210,47 +2294,214 @@
                     "term": "EU DCC",
                     "anchor": "eu_dcc",
                     "description": "Siehe <a href='#glossary_dcc'>DCC</a>"
+                },
+                {
+                    "term": "European Federation Gateway Service (EFGS)",
+                    "anchor": "efgs",
+                    "description": "Eine zentrale Infrastruktur, welche dazu dient <a href='#glossary_diagnosis_key'>Diagnoseschlüssel</a> über Ländergrenzen hinweg auszutauschen und zu prüfen."
+                },
+                {
+                    "term": "Event Registrierung",
+                    "anchor": "event_registration",
+                    "description": "Registrieren für eine Veranstaltung oder einen Besuch eines Ortes."
+                },
+                {
+                    "term": "Exposure Notification Express (ENE)",
+                    "anchor": "ene",
+                    "description": "Ab iOS 13.7 können Personen über eine mögliche Exposition gegenüber COVID-19 informiert werden, ohne dass eine spezielle App für Expositionsbenachrichtigungen erforderlich ist. Diese App-freie Funktion wird Exposure Notifications Express genannt und ist nur verfügbar, wenn eine Gesundheitsbehörde (PHA) sie unterstützt."
+                },
+                {
+                    "term": "Exposure Notification Framework (ENF)",

```

Comment:
Also from Apple.
Google uses ENS name not ENF.
See comment in English FAQ.

Reply:
Found a solution for that, see english comment